### PR TITLE
feat: add docs how create firebase project and fix mirror server docker error

### PR DESCRIPTION
**Pull request by Rybasher**

### CODE REVIEW

File: mirror-docs/docs/open-source-code/firebase.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+---
+
+# Creating a Firebase Project
+
+## Prerequisites
+
+Before you start, make sure you have:
+
+- A Google account
+- Internet access
```
Comment (Kluskey) [inline mirror-docs/docs/open-source-code/firebase.mdx]: Let's remove this internet access part

File: mirror-web-server/.env.example [hunk 2]
```diff
@@ -62,4 +62,5 @@ DISABLE_EMAIL=true
 ASSET_STORAGE_DRIVER=LOCAL
 
 # if ASSET_STORAGE_DRIVER is not GCP, then this is required 
-ASSET_STORAGE_URL=http://localhost:9000/
+ASSET_STORAGE_URL=http://localhost:9000/assets-storage/
```
Comment (Kluskey) [inline mirror-web-server/.env.example:65]: Where is this in the code? I checked but couldn't find it

**Review by Kluskey — APPROVED** [review]

File: mirror-docs/docs/open-source-code/firebase.mdx (hunk 1 above)
Comment (RevoluPowered) [inline mirror-docs/docs/open-source-code/firebase.mdx]:
```suggestion
```

**Review by RevoluPowered — APPROVED** [review]
LGTM

File: mirror-docs/docs/open-source-code/firebase.mdx (hunk 1 above)
Comment (RevoluPowered) [reply mirror-docs/docs/open-source-code/firebase.mdx]: i fixed this

File: mirror-web-server/.env.example (hunk 2 above)
Comment (Rybasher) [reply mirror-web-server/.env.example:65]: Here's an example of how the new path for asets looks like when using a local driver, I forgot to note this in the main PR
Also, how it looks like in the code
image
image

**Pull request merged**